src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,26 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  // eslint-disable-next-line object-curly-newline
+  const { completed, title, user, id } = todo;
+
+  return (
+    <article
+      className={classNames('TodoInfo', {
+        'TodoInfo--completed': completed,
+      })}
+      data-id={id}
+    >
+      <h2 className="TodoInfo__title">{title}</h2>
+
+      {!!user && <UserInfo user={user} />}
```

Comment:
[Operator Logical AND](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_AND) convert to boolean value
so you can write shorter `{user && <UserInfo user={user} />}`